README.md

```diff
@@ -17,6 +17,8 @@ This directory contains directly usable tools installable via:
 
 ```
 go get periph.io/x/cmd/...
+# From Go 1.17 onwards use the following - see https://go.dev/doc/go-get-install-deprecation
+go install periph.io/x/cmd/...
```

Comment:
I believe it's `go install periph.io/x/cmd/...@latest` ?

Reply:
The help talks about using "module aware" mode without it.  As I'm not 100% sure what that is, `@latest` seems like a good idea.  Update coming.  I was trying to make `codecov.yml` not complain, but the `ignore:` didn't seem to do it. Will remove.